cremona/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Would recommend adding `"strict": true,` but I'm guessing that may cause some issues.

Reply:
Thanks! I thought that was added already.